yaml/u2f.basic.flow.yaml

```diff
@@ -0,0 +1,24 @@
+title: U2F
+actor:
+-  D: Device
+-  B: Browser
+-  S: Server
+sequence:
+- B:
+    S:
+      arrow: "USERNAME and PASSWORD"
+      to: "Verify PASSWORD\nand generate CHALLENGE"
+- S:
+    B:
+      arrow: "$C$ = CHALLENGE"
+- B:
+    D:
+      arrow: "$C$"
+      to: "User touches button"
+- D:
+    B:
+      arrow: "$R$ = RESPONSE"
```

Comment:
Could you make it more transparent by showing the relationship of challenge and response?